fw/http_parser.c

```diff
@@ -864,35 +887,33 @@ process_trailer_hdr(TfwHttpMsg *hm, TfwStr *hdr, unsigned int id)
 		return CSTR_POSTPONE;					\
```

Comment:
I don't see the connection between `RGen_Hdr` and the `data` usage.

`RGen_Hdr` allows us to employ `&msg->stream->parser.hdr` which is implicitly used by `__msg_hdr_chunk_fixup`, but the `data` for inner FSM is set by `TFW_HTTP_PARSE_SPECHDR_VAL` and `TFW_HTTP_PARSE_RAWHDR_VAL` macroses, and in its turn the `data` modification is possible because all preceding chunks are already appended into `msg->stream->parser.hdr` by `tfw_http_parse_req` and `tfw_http_parse_resp`.